Spicy tuna/feature/image-upload

Spicytuna24 opened the pull request:
Completed image upload route

Commit:
SpicyTuna/feature/image-upload

Commit:
SpicyTuna/feature/image-upload

Commit:
SpicyTuna/feature/image-upload

PGrad commented on server/routes/treeimages/treeimages-router.js:
```diff
@@ -0,0 +1,47 @@
+  }
+
+  if (idImage && id) {
+    const responseImage = await getImageById(idImage && id);
```
@Spicytuna24 I'm a little confused, shouldn't you call `getImageById()` like this:
```ts
getImageById(idImage, id)
```
Assuming you can omit the filename from the arguments.

PGrad committed:
fix(photos api): fix the api routes

Making changes to the treeimages
router so it can accept requests
to upload and get images.

I use postman to test the routes.

The DB still needs to be set up,
so you can't actually add or get
images yet.
